backend/src/index.ts

```diff
@@ -384,6 +386,19 @@ handlers[API_ANIMATION_VIDEO] = async (input) => {
   return {response: new Response(result, responseOptions(CONTENT_TYPE_VIDEO_MP4))};
 };
 
+handlers[API_PROFILE_AVATAR] = async (input) => {
+  const result = await db.get(`profile/avatar:${expectUuidParam(input, "id")}`, "arrayBuffer");
+  // TODO how to handle content type?
+  return {response: new Response(result, responseOptions(CONTENT_TYPE_VIDEO_MP4))};
+};
+
+handlers[API_PROFILE_AVATAR_CREATE] = async (input) => {
+  const id = uuid();
```

Comment:
Rather than creating a new avatar with a new unique id each time, I think we should use
`const user = await input.requireAuthedUser();`

And then use `user.id` instead. Mainly because we know that the user can only have one avatar.

Reply:
Sounds good, I'll change to use the `user.id`